testing/scala/APKBUILD

```diff
@@ -0,0 +1,111 @@
+# Contributor: norbjd
+# Maintainer:
+pkgname=scala
+pkgver=2.12.2
+pkgrel=0
+pkgdesc="Scala 2.12.2"
```

Comment:
I'd go with a somewhat more descriptive description here, even just `The Scala programming language` should do.